UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/SalesCatalogueClient.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace UKHO.ExchangeSetService.Common.Helpers
+{
+    [ExcludeFromCodeCoverage] ////Excluded from code coverage as it has actual http calls 
+    public class SalesCatalogueClient : ISalesCatalogueClient
+    {
+        private readonly HttpClient httpClient;
+
+        public SalesCatalogueClient(HttpClient httpClient)
+        {
+            this.httpClient = httpClient;
+        }
+
+        public async Task<HttpResponseMessage> CallSalesCatalogueServiceApi(HttpMethod method, string postBody, string authToken, string uri)
```

Comment:
```suggestion
        public async Task<HttpResponseMessage> CallSalesCatalogueServiceApi(HttpMethod method, string requestBody, string authToken, string uri)
```

Reply:
Made changes related to property name in 6850 PBI PR